doc/examples/segmentation/plot_watershed.py

```diff
@@ -54,7 +54,7 @@
 
 ax0.imshow(image, cmap=plt.cm.gray, interpolation='nearest')
 ax0.set_title('Overlapping objects')
-ax1.imshow(-distance, cmap=plt.cm.jet, interpolation='nearest')
+ax1.imshow(-distance, cmap=plt.cm.gray, interpolation='nearest')
```

Comment:
I'd prefer to use non-grayscale colormap here, as it better presents what is actually the result.